exp/ingest/ledgerbackend/database_backend.go

```diff
@@ -46,6 +46,9 @@ func (dbb *DatabaseBackend) GetLatestLedgerSequence() (uint32, error) {
 	if err != nil {
 		return 0, errors.Wrap(err, "couldn't select ledger sequence")
 	}
+	if len(ledger) == 0 {
+		return 0, errors.New("No ledgers found in ledgerheaders table")
```

Comment:
Errors should not be capatlized for these reasons: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

It also might be less ambiguous to make the error state that the ledgerheaders table is empty or that there are no ledgerheaders rather than none were found, because it answers the question "why were none found?" and we know definitively at this point that that is the case.

```suggestion
		return 0, errors.New("no ledgers exist in ledgerheaders table")
```